protostar/commands/test/test_collector.py

```diff
@@ -16,17 +20,56 @@
 from protostar.protostar_exception import ProtostarException
 from protostar.utils.starknet_compilation import StarknetCompiler
 
+TestSuiteGlob = str
+TestSuitePath = Path
+TestCaseGlob = str
+Target = str
+"""e.g. `tests/**/::test_*`"""
+TestCaseGlobsDict = Dict[TestSuitePath, Set[TestCaseGlob]]
```

Comment:
We probably can skip this abstraction at all, as we dissused on Slack

Reply:
This is a short name for this dict. It affects the name of functions, parameters, variables and makes the name consistent. Renaming this type to more explicit form would make the code too explicit for the scope of this type.